stdlib/include/short_vec.isph

```diff
@@ -166,7 +166,7 @@ template <typename T, uint N> inline uniform T<N> select(uniform bool cond, unif
     uniform T<N> result;
 
     foreach (i = 0 ... N) {
-        result[i] = select(cond, t[i], f[i]);
+        result[i] = select((UIntMaskType)cond, t[i], f[i]);
```

Comment:
@nurmukhametov I am bit late here but shouldn't this have been `(uniform UIntMaskType)` instead? I expect the current one to work and to be equally fast on x86, but maybe not on other (future) targets.